pom.xml

```diff
@@ -94,8 +94,8 @@
     <pig.version>0.16.0</pig.version>
     <pig.classifier>h2</pig.classifier>
     <thrift-maven-plugin.version>0.10.0</thrift-maven-plugin.version>
-    <thrift.version>0.12.0</thrift.version>
-    <format.thrift.version>0.12.0</format.thrift.version>
+    <thrift.version>0.9.3</thrift.version>
```

Comment:
You shall not downgrade the thrift version.
BTW, this is the cause of the Travis failure. There is a rat exclusion for the thrift install directory which is currently `thrift-0.12.0` while the exclusion contains `thrift-0.9.3` because this modification.